TeamCode/src/main/java/org/firstinspires/ftc/teamcode/auto/ForwardOnly.java

```diff
@@ -0,0 +1,28 @@
+package org.firstinspires.ftc.teamcode.auto;
```

Comment:
Do we still need this? Is this for parking in the corner?

Reply:
Might as well have it